hw/ip/i2c/doc/_index.md

```diff
@@ -33,7 +33,7 @@ See that document for integration overview within the broader top level system.
 - Direct SCL and SDA control in "Override mode" (for debugging)
 - SCL and SDA ports mapped to I/O via the pinmux.
 - Interrupts in the host mode for FMT and RX FIFO overflow, target NACK, SCL/SDA signal interference, timeout, unstable SDA signal levels, and transaction complete
-- Interrupts in the target mode for TX FIFO empty during a read, TX FIFO nonempty at the end of a read, TX and ACQ FIFO overflow, host sending STOP after ACK, and host ceasing to send SCL pulses during an ongoing transaction
+- Interrupts in the target mode for TX FIFO empty during a read, TX FIFO nonempty at the end of a read, TX overflow and ACQ FIFO full, host sending STOP after ACK, and host ceasing to send SCL pulses during an ongoing transaction
```

Comment:
TX overflow and FMT overflow both seem weird to me. I still can't figure out why they exist when it's easier to check the level before writing than to service the interrupt handler. I'm also concerned that overflowing the FMT buffer could hang the peripheral by overwriting a START or STOP symbol.

Didn't transaction_complete get added here too?

Reply:
ah trans_complete was done in a previous PR.